tests/locale/keymap_or_locale_x11.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Keyboard layout test in console and display manager after boot
+# Maintainer: Martin Loviska <mloviska@suse.com>
```

Comment:
@mloviska I understood you do not want to stay as maintainer, should we update that to e.g. @Soulofdestiny or me?

Reply:
done